### PR TITLE
Library Updates

### PR DESCRIPTION
**Pull request by gallifreyancat**
Added reverse gyro drive and color sensors to CrabRobotLibrary.  Added movement to KateSkillLibrary.

### CODE REVIEW

**Review by codefeld — COMMENTED** [review]
Nice job adding the color sensors! I saw that you added a reverse gyro drive function. We already have one called "bw_gyro_drive" that we are using. I would suggest taking out the one you added because having two reverse gyro drive functions may become confusing for some people.

File: CrabRobotLibrary.py [hunk 1]
```diff
@@ -14,6 +14,8 @@ class Robot:
 	brick = EV3Brick()
 	left_wheel = Motor(Port.A)
 	right_wheel = Motor(Port.D)
+	color_sensor_left = ColorSensor(Port.S1)
+	color_sensor_right = ColorSensor(Port.S4)
```
Comment (codefeld) [inline CrabRobotLibrary.py:18]: It looks like Zac added these same lines, so there is a conflict that needs to be resolved.

**Review by HipsterDave — APPROVED** [review]

**Review by HipsterDave — CHANGES_REQUESTED** [review]
Hmm... seems like this PR is conflicting with the master branch. If it is an outdated branch, I can help you bring it up to date!

**Comment by gallifreyancat** [comment]
Closed

**Pull request closed without merging**